lib/ansible/modules/copy.py

```diff
@@ -238,10 +239,10 @@
     type: str
     sample: 2a5aeecc61dc98c4d780b14b330e3282
 checksum:
-    description: SHA1 checksum of the file after running copy.
+    description: SHA256 checksum of the file after running copy.
```

Comment:
Seems to be a breaking API change. All existing playbooks will suddenly stop matching the values that the authors put in their checks, over the years.
I'd perhaps introduce a new additional result or additional input to select a checksum.
I see there's an `md5sum` above, so there's prior art somewhere around.